img/calendar-white.svg

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
I agree, but please use the optimized version - either copy from my PR or run scour command:
`scour -i calendar-white.svg -o calendar-white-new.svg --create-groups --enable-id-stripping --enable-comment-stripping --shorten-ids --remove-metadata --strip-xml-prolog --no-line-breaks`